projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-css-nudge-customize-control.php

```diff
@@ -42,7 +42,7 @@ class CSS_Nudge_Customize_Control extends \WP_Customize_Control {
 	public function render_content() {
 		$cta_url           = $this->cta_url;
 		$nudge_copy        = $this->nudge_copy;
-		$nudge_button_copy = __( 'Upgrade Now', 'jetpack' );
+		$nudge_button_copy = __( 'Upgrade now', 'jetpack' );
```

Comment:
This is very minor but the preference is now title case and that's what happens with all the other buttons across WP.com/Jetpack

Reply:
This is a good idea, good catch!